opentracing-noop/src/main/java/io/opentracing/NoopSource.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2016-2017 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing;
+
+import java.util.Map;
+
+/**
+ * A noop (i.e., cheap-as-possible) implementation of a Source.
+ */
+public class NoopSource implements ActiveSpanSource {
```

Comment:
I did something here... @pavolloffay @cwe1ss let me know if you had something else in mind. I wanted to avoid multiple files for the noop stuff, so it's a bit condensed.